conf/modules.config

```diff
@@ -26,165 +26,237 @@ process {
-                "-variable_modifications ${params.variable_mods.tokenize(',').collect {"'${it}'"}.join(" ")}",
-                "-enzyme '${params.enzyme}'",
-                "-spectrum_batch_size ${params.spectrum_batch_size}"
-            ].join(' ').trim()
-            publishDir = [
-                path: {"${params.outdir}/intermediate_results/comet"},
-                mode: params.publish_dir_mode,
-                pattern: '*.tsv'
-            ]
-        }
+        ext.args    = [
+            "-precursor_mass_tolerance ${params.precursor_mass_tolerance}",
+            "-fragment_mass_tolerance ${params.fragment_mass_tolerance}",
+            "-fragment_bin_offset ${params.fragment_bin_offset}",
+            "-instrument ${params.instrument}",
+            "-num_hits ${params.num_hits}",
+            "-digest_mass_range ${params.digest_mass_range}",
+            "-max_variable_mods_in_peptide ${params.number_mods}",
+            "-missed_cleavages 0",
+            "-precursor_charge ${params.prec_charge}",
+            "-activation_method ${params.activation_method}",
+            "-variable_modifications ${params.variable_mods.tokenize(',').collect {"'${it}'"}.join(" ")}",
+            "-enzyme '${params.enzyme}'",
+            "-spectrum_batch_size ${params.spectrum_batch_size}"
+        ].join(' ').trim()
+        publishDir = [
+            path: {"${params.outdir}/intermediate_results/comet"},
+            mode: params.publish_dir_mode,
+            pattern: '*.tsv'
+        ]
+    }
 
-    withName: 'OPENMS_IDFILTER_FOR_ALIGNMENT' {
-            ext.args    = [
-                "-remove_decoys",
-                "-precursor:length '${params.peptide_min_length}:${params.peptide_max_length}'",
-                "-delete_unreferenced_peptide_hits",
-                (params.fdr_threshold == '0.01') ? "-score:pep 0.05" : "-score:pep " + params.fdr_threshold
-            ].join(' ').trim()
-            publishDir  = [
-                path: {"${params.outdir}/intermediate_results/alignment"},
-                mode: params.publish_dir_mode,
-                pattern: '*.idXML'
-            ]
-        }
+    withName: 'OPENMS_PEPTIDEINDEXER' {
+        publishDir  = [
+            enabled: false
+        ]
+    }
 
     withName: 'OPENMS_IDFILTER_Q_VALUE' {
-            ext.prefix  = {"${meta.id}_fdr_filtered"}
-            ext.args    = [
-                "-remove_decoys",
-                "-precursor:length '${params.peptide_min_length}:${params.peptide_max_length}'",
-                "-delete_unreferenced_peptide_hits",
-                (params.fdr_threshold == '0.01') ? "-score:pep 0.05" : "-score:pep " + params.fdr_threshold
-            ].join(' ').trim()
-            publishDir  = [
-                path: {"${params.outdir}/intermediate_results/alignment"},
-                mode: params.publish_dir_mode,
-                pattern: '*.idXML'
-            ]
-        }
+        ext.prefix  = {"${meta.id}_pout_filtered"}
```

Comment:
Why `pout` instead of `out`

Reply:
I thought it is more intuitive for the user. We have the `pin` file, which stands for percolator-in file and that is frequently used for percolator input, so pout made sense to me..

Reply:
maybe easier to store the output under a percolator folder instead of annotation it with `pout` 🤔

Reply:
I store now both, pout and pout_filtered in percolator folder since sometimes it is good to have the unfiltered percolator output. Imo nice to have the annotation otherwise one can get confused with all kinds of files..